eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -32,16 +32,23 @@ import grizzled.slf4j.Logging
 import scodec.bits.BitVector
 import scodec.codecs._
 import scodec.{Attempt, Codec}
-import shapeless.{HNil, ::}
```

Comment:
nit: there are other unused imports you can clean up

Reply:
Done with b90a9e078.